packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -119,33 +208,16 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
         right: 0,
         bottom: 0
       },
-      !isOpen &&
-        !isAnimating &&
-        isHiddenOnDismiss && {
-          visibility: 'hidden'
-        },
       isCustomPanel && classNames.custom,
       className
     ],
     overlay: [
       {
-        pointerEvents: 'none',
-        opacity: 1,
-        cursor: 'pointer',
-        transition: `opacity ${AnimationVariables.durationValue3} ${AnimationVariables.easeFunction1}`,
```

Comment:
This `transition` is doing the same thing as the Animation classes below in a better way